config/asseco-authentication.php

```diff
@@ -4,7 +4,7 @@
     /**
      * Url the voice:fetch-key command will call to fetch the public key.
      */
-    'ima_uri'                    => env('IAM_URI'),
+    'iaa_uri'                    => env('IAM_URI'),
```

Comment:
Hahah, jel ti mene zajebavaš? :D

Reply:
hahahahah 

I dont want to talk about it :(